src/Microsoft.Dnx.ApplicationHost/DefaultHost.cs

```diff
@@ -183,6 +185,22 @@ private void Initialize(RuntimeOptions options, IServiceProvider hostServices, I
             AddBreadcrumbs(libraries);
         }
 
+        private void ValidateMinRuntimeVersion(IEnumerable<LibraryDescription> libraries)
+        {
+            foreach (var lib in libraries)
```

Comment:
We can avoid the 2 for loops and do the servicing and this in a single pass but that's not important right now.